content/snowplow-recommendations/_index.md

```diff
@@ -0,0 +1,83 @@
+
+
+
+In this accelerator, we will enable you to:
+
+* Run dbt to transform your Snowplow data into a format that can be used by AWS Personalize
+* Set up the infrastructure required to run AWS Personalize, including the necessary IAM roles and policies
+* Create ecommerce and/or media (video_on_demand) recommenders in AWS Personalize
+* Run a local Flask application to interact with AWS Personalize 
+***
+
+#### Who is this guide for?
+
+- Data practitioners who are already familiar with dbt and Python, as well as running commands in the command line.
+- Data practitioners who want to use AWS Personalize to create recommenders for either ecommerce or media (video_on_demand) based on their Snowplow data.
+- Data practitioners who would like to learn with a hands-on approach about AWS Personalize and how it can be used to create recommenders, using sample data.
+***
+
+#### What will be covered
+
+In an estimated minimum of 5.5 hours, you can achieve the following:
+
+- **Upload -** Upload some sample data (optional)
+- **Create Supporting Infrastructure -** Create the supporting infrastructure needed move your data from your warehouse to S3, including the necessary IAM roles and policies
+- **Model -** Run dbt to transform your Snowplow data into a format that can be used by AWS Personalize
+- **Create Recommenders -** Create recommenders in AWS Personalize
+- **Interact with Recommenders -** Run a local Flask application to interact with AWS Personalize
+- **Next Steps -** Learn about the next steps you can take to continue your journey with AWS Personalize
+***
+
+#### What will not be covered
+
+- We will not be covering how to implement ecommerce or media tracking in this accelerator. We will also not be covering how to run the ecommerce or media dbt packages. If you would like to learn more about these, the [quickstart](https://docs.snowplow.io/docs/modeling-your-data/modeling-your-data-with-dbt/dbt-quickstart/ecommerce/) or [accelerator](https://docs.snowplow.io/accelerators/ecommerce/) for ecommerce or the [quickstart](https://docs.snowplow.io/docs/modeling-your-data/modeling-your-data-with-dbt/dbt-quickstart/media-player/) for media could be a good place to start (media accelerator coming soon).
+
+<!-- TODO fix gantt chart styling - Section names overlap chart -->
+{{<mermaid>}} 
+gantt
+        dateFormat  HH:mm
+        axisFormat %H:%M
+        section 1. Upload
+        30min       :s1, 00:00, 00:30
+        section 2. Create Supporting Infrastructure
+        2h          :s2, after s1, 02:30
+        section 3. Model
+        30min       :s3, after s2, 03:00
+        section 4. Create Recommenders
+        1h          :s4, after s3, 04:00
+        section 5. Interact with Recommenders
+        1h          :s5, after s4, 05:00
+        section 6. Next steps
+        30min       :s6, after s5, 05:30
+{{</mermaid>}}
```

Comment:
unrelated to this PR, but Jack do you have any idea how to make this chart look better, so that the text doesn't go into the chart area? I tried when I was writing it but couldn't figure it out, and it looks kinda janky